binder/environment.yml

```diff
@@ -35,6 +35,7 @@ dependencies:
 - qtconsole >=4.6.0
 - qtpy >=1.5.0
 - rtree >=0.8.3
+# - toposort >=1.5
```

Comment:
@ccordoba12, toposort is not available on anaconda, defaults

I could bundle it in the meantime?

It is a single file

https://bitbucket.org/ericvsmith/toposort/src/default/toposort.py but with http://www.apache.org/licenses/LICENSE-2.0

Any suggestions?